packages/loader/src/PrimitiveMeshLoader.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  AssetPromise,
+  AssetType,
+  LoadItem,
+  Loader,
+  ModelMesh,
+  PrimitiveMesh,
+  resourceLoader
+} from "@galacean/engine-core";
+
+@resourceLoader(AssetType.PrimitiveMesh, ["mesh"], false)
+class PrimitiveMeshLoader extends Loader<ModelMesh> {
+  load(item: LoadItem, { engine }): AssetPromise<ModelMesh> {
+    return this.request<IPrimitiveMesh>(item.url, {
+      ...item,
+      type: "json"
+    }).then((data) => {
+      switch (data.type) {
+        case PrimitiveMeshType.Sphere:
+          return PrimitiveMesh.createSphere(engine, data.sphereRadius, data.sphereSegments);
```

Comment:
Use `createSubdivisionSurfaceSphere`, `createSphere` will deprecate, don't need to support anymore

Reply:
done